components/esp_lvgl_port/examples/touchscreen/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# The following lines of boilerplate have to be in your project's CMakeLists
+# in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.16)
+
```

Comment:
Could you please add this line?
```
# "Trim" the build. Include the minimal set of components, main and anything it depends on.
set(COMPONENTS main)
```

By default, all esp-idf components are built. With this line added, only components required by main are built